build.sbt

```diff
@@ -139,9 +175,24 @@ lazy val telnetd = project
   .in(file("modules/telnetd"))
   .dependsOn(service, sql)
   .enablePlugins(JavaAppPackaging)
+  .enablePlugins(DockerPlugin)
   .settings(commonSettings)
   .settings(resolvers += "bmjames Bintray Repo" at "https://dl.bintray.com/bmjames/maven")
   .settings(
     libraryDependencies += "org.tpolecat" %% "tuco-core" % "0.1.0",
-    dockerExposedPorts  := List(6666)
+    dockerExposedPorts  := List(6666),
+    dockerRepository    := Some("geminihlsw")
+  )
+
+lazy val ctl = project
+  .in(file("modules/ctl"))
+  .settings(commonSettings)
+  .settings (
+    resolvers += "bmjames Bintray Repo" at "https://dl.bintray.com/bmjames/maven",
+    libraryDependencies ++= Seq(
+      "org.scalaz"  %% "scalaz-core"   % scalazVersion,
+      "org.scalaz"  %% "scalaz-effect" % scalazVersion,
+      "net.bmjames" %% "scala-optparse-applicative" % "0.5"
```

Comment:
`gemctl` takes command-line arguments and switches and stuff, and this library provides a parser that also composes the help message you see when you do `gemctl deploy -h` for instance. It's also used by the telnet server.

It's a small totally pure lib and I'm a maintainer so I'm not too worried about pulling it in. We're going to port it to cats pretty soon.